.rustfmt.toml

```diff
@@ -10,15 +10,46 @@ reorder_imports = true
 reorder_modules = true
 tab_spaces = 4
 use_field_init_shorthand = true
+use_small_heuristics = "Default"
 use_try_shorthand = true
-edition = "2018"
 
-# unstable options (for one-off fixes and CI)
-merge_imports = true
-wrap_comments = true
+# unstable options
+binop_separator = "Front"
+blank_lines_lower_bound = 0
+blank_lines_upper_bound = 1
+brace_style = "PreferSameLine"
+combine_control_expr = false
```

Comment:
`true` is the default value.
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#combine_control_expr

I changed this because in the examples they show, `false` led to code that was easier to read, imo.

Reply:
Sure, sounds good.